cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -282,7 +284,7 @@ These are the available commands:
    **Explore**
    ``execute <query> [<timeout>]``,"Executes a Query with optional <timeout> in minutes (default is no timeout)."
    **Ingest**
-   ``load stream <stream-id> <local-file-path> [<content-type>]``,"Loads a file to a Stream. The contents of the file will become multiple events in the Stream, based on the content type. If <content-type> is not provided, it will be detected by the file extension."
+   ``load stream <stream-id> <local-file-path> [<content-type>]``,"Loads a file to a Stream. The contents of the file will become multiple events in the Stream, based on the content type (avro/binary, text/csv, text/plain, text/tsv, text/plain). If <content-type> is not provided, it will be detected by the file extension. Supported file extensions: avro, csv, log, tsv, txt."
```

Comment:
you have text/plain twice...so  you need the unique values of the keys in the map